templates/download/index.twig

```diff
@@ -27,116 +24,104 @@
+		<div id="prerelease" class="text-center">
+			<small>
+				<span class="fas fa-exclamation-circle"></span>
+				{% trans %}Downloads labeled Alpha, Beta, or Nightly are pre-release software, stability may suffer.{% endtrans %}
+			</small>
+		</div>
 	{% endif %}
+	{{ trailer|raw }}
 </div>
+{% endmacro %}
 
-<div id="mac-div" class="text-center hidden container">
-	<h2>{% trans %}Install LMMS on macOS{% endtrans %}</h2>
-	<p>{% trans %}Click one of the buttons below to download LMMS for macOS{% endtrans %}</p>
-	{% if osxstable %}
-		<h3>{% trans %}Stable Versions{% endtrans %}</h3>
-		{% for osx in osxstable %}
-			{{ download.printrelbutton(osx) }}
-		{% endfor %}
-		{{ download.releasenotes(osxstable[0]['release']['body'], 'osxstable') }}
-	{% endif %}
-	{% if osxpre %}
-		<h3>
-		{% if 'alpha' in osxpre[0].release.tag_name %}
-			{% trans %}Nightly Versions{% endtrans %}
-		{% else %}
-			{% trans %}Beta Versions{% endtrans %}
-		{% endif %}
-		</h3>
-		{% for osx in osxpre %}
-			{{ download.printrelbutton(osx, 'btn-warning') }}
-		{% endfor %}
-		{{ download.releasenotes(osxpre[0]['release']['body'], 'osxpre') }}
-	{% endif %}
+{% import 'macros.twig' as macros %}
+
+{% block content %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+{{ macros.jumbo('Download LMMS'|trans,
+'Downloading and using LMMS is free! Just choose the operating system you want to run LMMS on:'|trans) }}
+<div class="text-center container">
+	<div>
+		<div class="btn-group" data-toggle="buttons">
+			{{ _self.osbutton('linux', 'fa-linux', 'Linux') }}
+			{{ _self.osbutton('windows', 'fa-windows', 'Windows') }}
+			{{ _self.osbutton('mac', 'fa-apple', 'macOS') }}
+		</div>
+	</div>
 </div>
 
-<div id="prerelease" class="text-center"><small><span class="fas fa-exclamation-circle"></span> {% trans %}Downloads labeled Beta or Nightly are pre-release software, stability may suffer.{% endtrans %}</small></div>
-<hr><div class="text-center spacey"><a class="btn btn-default" href="https://github.com/LMMS/lmms/tags"><i class="fas fa-history"></i> {% trans %}Previous releases{% endtrans %}</a></div>
+{{ _self.ospage(
+	'linux',
+	'Install LMMS on Linux'|trans,
+	'Click one of the buttons below to download LMMS for Linux.'|trans,
+	linstable,
+	linpre,
+	linnightly,
+	'Run <code>chmod +x ~/Downloads/*.AppImage</code> to set the <code>.AppImage</code> executable before running'|trans,
+	include('download/linux.twig')
+) }}
+
+{{ _self.ospage(
+	'windows',
+	'Install LMMS on Windows'|trans,
+	'Click one of the buttons below (either 32bit or 64bit) to download LMMS for Windows'|trans,
+	winstable,
+	winpre,
+	winnightly
+) }}
+
+{{ _self.ospage(
+	'mac',
+	'Install LMMS on macOS'|trans,
+	'Click one of the buttons below to download LMMS for macOS'|trans,
+	osxstable,
+	osxpre,
+	osxnightly
+) }}
+
+<div class="text-center spacey container">
+	<hr>
```

Comment:
It had zero width, since browsers give it auto left and right margins by default, which doesn't play well with a containing flex element. I moved it inside that div, which allowed it to take up the correct width again. It could also be fixed with some CSS, but I was lazy and didn't want to set up SCSS compilation on Windows.